02-peer-protocol.md

```diff
@@ -366,6 +366,12 @@ This message introduces the `channel_id` to identify the channel. It's derived f
 
 #### Requirements
 
+Both peers:
```

Comment:
I think we may also want to define a new bit in the funding flags within the `OpenChannel` message to allow nodes to modulate this value, since there'll be optional _and_ required feature bits for it. It'll take some time for channels to switch over (since I don't foresee the entire networking shutting down and transitioning to this new channel type). For some period of time, implementations will need to understand both the old and new commitment types, and a node can't feasible _only_ use this new channel type. 

If we add a new funding flag, then we'll be able to easily flip it via a config option or the like to ensure we all still understand the old and new formats. This will also be useful for new implementations as they'll still need to interface with the rest of the legacy network. 

In short I propose two signalling venues:
   * the global feature bit
   * a new defined in in the `open_channel` message's funding flags

Reply:
I think current feature negotiation is sufficient.  The open_channel flag seems completely redundant?